src/praisonai/praisonai/cli/legacy/direct_prompt.py

```
@@ -0,0 +1,1280 @@
+                    print(f"[green]Image generated successfully![/green]")
+                    if 'url' in result['data'][0]:
+                        print(f"URL: {result['data'][0]['url']}")
+            
+            return result
+        
+        # CLI Backend - delegate agent turns to external CLI tools
+        if hasattr(self, 'args') and getattr(self.args, 'cli_backend', None):
+            try:
+                from praisonai_code.cli_backends import resolve_cli_backend
+                agent_config["cli_backend"] = resolve_cli_backend(self.args.cli_backend)
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)